voyage-model-provider/main.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+
+from voyageai import AsyncClient
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+
+debug = os.environ.get("GPTSCRIPT_DEBUG", "false") == "true"
+client = AsyncClient(api_key=os.environ.get("OTTO8_VOYAGE_MODEL_PROVIDER_API_KEY", "pa-P3XFfrvvOj1hzJSdUmkV_O7RVngUQ-00HpsDzgRX8sc"))
+app = FastAPI()
+uri = "http://127.0.0.1:" + os.environ.get("PORT", "8000")
+
+voyage_models = [
+    {"id": "voyage-3"},
+    {"id": "voyage-3-lite"},
+    {"id": "voyage-finance-2"},
+    {"id": "voyage-multilingual-2"},
+    {"id": "voyage-law-2"},
+    {"id": "voyage-code-2"},
+]
+
+
+def log(*args):
+    if debug:
+        print(*args)
+
+
+@app.middleware("http")
+async def log_body(request: Request, call_next):
+    body = await request.body()
+    log("HTTP REQUEST BODY: ", body)
+    return await call_next(request)
+
+
+@app.get("/")
+@app.post("/")
+async def root():
+    return uri
+
+
+@app.get("/v1/models")
+async def list_models() -> JSONResponse:
+    return JSONResponse({"data": [{"id": voyage_models}]}, status_code=200)
```

Comment:
Looks like this might be slightly off:
```suggestion
    return JSONResponse({"data": voyage_models}, status_code=200)
```

Reply:
Good catch. Thanks.